src/components/TextInput/BaseTextInput.js

```diff
@@ -200,6 +200,7 @@ class BaseTextInput extends Component {
         const inputHelpText = this.props.errorText || this.props.hint;
         const formHelpStyles = this.props.errorText ? styles.formError : styles.formHelp;
         const placeholder = (this.props.prefixCharacter || this.state.isFocused || !hasLabel || (hasLabel && this.props.forceActiveLabel)) ? this.props.placeholder : null;
+        const disableAutoCorrectIfSecureText = this.props.secureTextEntry && ({autoCorrect: false});
```

Comment:
```suggestion
```

Reply:
hmm @parasharrajat, I did try this one before I remembered this has weird issues like non-secureTextEntry will always apply autoCorrect to true below is an example whereby it uses autoCorrect to phone/email inputs currently we do not have autoCorrect to it but after adding the above suggestion it will have.

**After this suggestion**
<img src=https://user-images.githubusercontent.com/47522946/194464609-8879ee3f-7cba-4f36-b0dc-9ca441687d5c.png  width=50%>


**Without this suggestion**
<img src=https://user-images.githubusercontent.com/47522946/194464619-a61ffc06-9267-4cf2-a42e-d819c178c9d2.png  width=50%>

Reply:
Did you tests my other suggestions as well?

Reply:
Yup other all suggestions are added in Pr

Reply:
Yup other all suggestions are added in Pr

Reply:
let me check

Reply:
sure

Reply:
Continuing on this correct thread.